### PR TITLE
Add empfangsbevollmaechtigter

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- We discovered that we need more fields, namely the Empfangsvollmacht fields. This adds them.

# Changes
- Add fields to input
- Add output xml objects
- Adapt tests

# Feedback
- None specific

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

**Comment by frederike-ramin** [comment]
The duplicated code that SonarCloud finds is merely in tests, which shouldn't be a problem and doesn't have any value in changing, So I'll merge regardless once I have an approve^^

**Review by shrzaf — APPROVED** [review]

**Pull request merged**
